libraries/standard/serializer/include/iot_serializer.h

```diff
@@ -275,12 +289,14 @@ typedef struct IotSerializerEncoderObject
 typedef struct IotSerializerDecoderObject
 {
     IotSerializerDataType_t type;
+    const uint8_t * bufferPtr;
```

Comment:
With our naming convention, it should be pBuffer

Reply:
Done